vendor/github.com/prometheus/alertmanager/cluster/cluster.go

```diff
@@ -142,6 +142,7 @@ func Create(
 	probeInterval time.Duration,
 	tlsTransportConfig *TLSTransportConfig,
 	allowInsecureAdvertise bool,
+	label string,
```

Comment:
@JohnnyQQQQ do you think it's worth bringing this into Mimir? I can't remember the exact set of conditions under which you faced this problem for the Alertmanager.

Reply:
This function doesn't seem to be used by Mimir. @gotjosh confirmed that privately: "OH right — we completely removed support for gossip-based alertmanager cluster. Memberlist in only used in the ring which already supports this."

Reply:
100% correct.